### DISCUSSION TITLE
mrgsolve.rdb is corrupt

### DISCUSSION THREAD

**Discussion by pharmdwoman** [metrumresearchgroup/mrgsolve#1141] [General]
Hi,
The mrgsolve package on my MacBook was working fine until recently, not sure what triggered it, I could not load the package and kept getting the error message below. I removed the package and re-installed via https://mpn.metworx.com/snapshots/stable/2022-02-11  but still got the same message. Any ideas why? 
Thanks,
Thuy

>   library(mrgsolve)
Error: package or namespace load failed for ‘mrgsolve’ in get(Info[i, 1], envir = env):
 lazy-load database '/Users/xxxxxx/Applications/R_libs/mpn_snapshot_2021-09-11/mrgsolve/R/mrgsolve.rdb' is corrupt
In addition: Warning message:
In get(Info[i, 1], envir = env) : internal error -3 in R_decompress1

**Comment by kyleam** [comment]
I'm not sure what the issue is but...

> I removed the package and re-installed via https://mpn.metworx.com/snapshots/stable/2022-02-11 

... the date shown in the message "mpn_snapshot_2021-09-11/mrgsolve/R/mrgsolve.rdb" doesn't match the date you reinstalled from.  Are you sure you successfully removed the package?

**Comment by kylebaron** [comment]
Hi @pharmdwoman  -

Now sure how you did the removal the first time; can you try navigating to `/Users/xxxxxx/Applications/R_libs/mpn_snapshot_2021-09-11/` and manually removing the mrgsolve installation?

Kyle

**Comment by pharmdwoman** [comment]
Hi @kylebaron,
Sorry, I named the personal folder with the 2021-09-11 date but I did make sure that the package had been uninstalled (I did it via RStudio). When I re-installed I used repos="https://mpn.metworx.com/snapshots/stable/2022-02-11" and I still got the same error message that the file is corrupt. 
Thuy

P.S. I'm using RStudio Version 2023.03.0+386 (2023.03.0+386) and R version 4.1.2, macOS Ventura 13.4.1
